Makefile

```diff
@@ -104,6 +104,8 @@ GOBIN := $(shell $(GO) env GOBIN)
 ifeq ($(GOBIN),)
 GOBIN := $(FIRST_GOPATH)/bin
 endif
+# Required by some scripts
+export GOBIN
```

Comment:
can you do this export  if it's not defined?  CI tests aren't happy.

Reply:
removed